controllers/api/userRoutes.js

```diff
@@ -86,4 +86,14 @@ router.post("/login", (req, res) => {
     });
 });
 
+router.post("/profile", tokenAuth, (req, res) => {
+ User.findByPk(req.user.id).then(foundUser=>{
+   res.json(foundUser);
+ })
+});
+
+router.get("/validateToken", tokenAuth, (req,res) => {
```

Comment:
the token validation is done by the tokenAuth function for each route, so we don't need this validation route